src/protocol/configuration.jl

```diff
@@ -7,8 +7,15 @@ end
     event::WorkspaceFoldersChangeEvent
 end
 
-@dict_readable struct DidChangeConfigurationParams
+struct DidChangeConfigurationParams
     settings::Any
+    function DidChangeConfigurationParams(d)
```

Comment:
No reason for this to be an inner constructor, right?

Reply:
I couldn't find a terse way to stop it recursing on itself as an outer constructor

Reply:
Ah yeah, of course.